revscoring/dependent.py

```diff
@@ -12,9 +12,11 @@ def __init__(self, name, process, dependencies=None):
         self.name = name
         self.process = process
         self.dependencies = dependencies if dependencies is not None else []
+        self.calls = 0
     
     def __call__(self, *args, **kwargs):
         logger.debug("Executing {0}.".format(self))
+        self.calls += 1
```

Comment:
Is there any chance of this hitting the max allowed integer (is there such a thing in Python?) when processing a huge dataset? Would that cause an error?